sky/backends/backend_utils.py

```diff
@@ -1861,7 +1861,7 @@ def _update_cluster_status(
         return global_user_state.get_cluster_from_name(cluster_name)
 
 
-def _refresh_cluster_record(
+def refresh_cluster_record(
```

Comment:
Maybe we should keep it private for now?

Reply:
Oops, that was a remnant. Fixed. Thanks!